config/locales/es/general.yml

```diff
@@ -449,6 +449,7 @@ es:
       title_external_url: "Documentación adicional"
       title_video_url: "Vídeo externo"
       author: Autor
+      proposals: Propuestas
```

Comment:
Do we need this proposals string? If so it should be present as well on the english translation file, or crowdin will delete this key

Reply:
That string is not needed. I deleted it and instead I use those one in the layous header section.